network/roles/nvidia_doca/vars/main.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+---
+
+# Usage: main.yml
+os_supported_leap: "leap"
+os_supported_rocky: "rocky"
+os_supported_rhel: "redhat"
+
+doca_repo_url: "https://linux.mellanox.com/public/repo/doca/{{ nvidia_doca_version }}/rhel/{{ compute_os_version }}/x86_64"
```

Comment:
correct URL example is `https://linux.mellanox.com/public/repo/doca/2.5.0/rhel8.0/x86_64/` 
please replace `rhel` with variable so can be used with other distros...